openshift/ansiblegen/docstrings.py

```diff
@@ -13,26 +12,49 @@
 import ruamel.yaml
 from ruamel.yaml.comments import CommentedMap
 
-from openshift import __version__ as openshift_version
-from openshift.client import models
-from openshift.helper import OpenShiftException
-from openshift.helper.ansible import AnsibleModuleHelper
+from kubernetes.client import models as k8s_models
 
-logger = logging.getLogger(__name__)
+from .. import __version__ as openshift_client_version
+from .. import __k8s_client_version__ as k8s_client_version
+from ..client import models as openshift_models
+from ..helper.exceptions import KubernetesException
+from ..helper.ansible import KubernetesAnsibleModuleHelper, OpenShiftAnsibleModuleHelper
 
+logger = logging.getLogger(__name__)
 
 # Once the modules land in Ansible core, this should not change
+# TODO: compute this by inspecting the Ansible repo
 ANSIBLE_VERSION_ADDED = "2.3.0"
 
 
 class DocStrings(object):
 
-    def __init__(self, model, api_version):
+    def __init__(self, model, api_version, is_openshift):
         self.model = model
         self.api_version = api_version
+
+        if is_openshift:
```

Comment:
I like that idea, but I think I'll defer to @chouseknecht to see his thoughts, since ansiblegen has been his baby.